1-exercises/B-setting-values/exercise2.js

```diff
@@ -26,6 +27,12 @@ let student = {
 */
 
 // write code here
+function isPassed (attendance, examScore) {
+  if ((attendance >= 90) && (examScore > 60)) {
+    return true;
```

Comment:
you could possibly write like:
function isPassed (attendance, examScore) {
 return attendance >= 90 && examScore > 60
}
'return' returns boolean!